src/Bundle.js

```diff
@@ -453,4 +464,55 @@ export default class Bundle {
 
 		return ordered;
 	}
+
+	trace ( module, localName, es6 ) {
+		const importDeclaration = module.imports[ localName ];
```

Comment:
Hmm... that's true, re-exports aren't supposed to create a binding in the local scope. I *think* it could be solved fairly easily by having a `module.reexports` alongside `module.imports` and `module.exports`, and using that in `bundle.trace()` and `module.mark()`, but that's just speculation at this point